schema.sql

```diff
@@ -47,3 +47,22 @@ ADD CONSTRAINT animals_constraint_two
 FOREIGN KEY (owners_id)
 REFERENCES owners (id)
 ON DELETE CASCADE;
+
+CREATE TABLE vets(
+  id INT GENERATED ALWAYS AS IDENTITY,
+  name VARCHAR(100),
+  age INT,
+  date_of_graduation DATE,
+  PRIMARY KEY(id)
+);
+
+CREATE TABLE specializations(
+  name VARCHAR(100),
+  specialties VARCHAR(100)
+);
+
+CREATE TABLE visits(
+  animal_name VARCHAR(100),
+  vet_name VARCHAR(100),
```

Comment:
- [ ] Just like I suggest above, let's try to use primary keys here.

See example:
```
CREATE TABLE visits (
   animal_id int,  -- a reference to an animal primary key
   vet_id int, -- a reference to a vet primary key
);
```

Reply:
- [ ] As suggested by @mmsesay, Kindly update the columns to use ids instead of names. This is because there may be more than one species or vet sharing the same name.  I have seen that you wrote a query to add an id column, instead, you should update the existing columns to use id.